pkg/controller/shoot/shoot_control.go

```diff
@@ -80,6 +104,39 @@ func (c *Controller) shootDelete(obj interface{}) {
 	c.getShootQueue(obj).Add(key)
 }
 
+func (c *Controller) reconcileConfigMapKey(key string) error {
+	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	if err != nil {
+		return err
+	}
+	logger.Logger.Infof("[ConfigMap Watcher] config map %v/%v was updated", namespace, name)
```

Comment:
Let's please not log this, we are watching all configmaps in the cluster and this would pollute the log.

Reply:
What do you think about to switch it to `Debugf`?

Reply:
Not sure, we only care about the ConfigMaps that are referenced by shoots. Therefore, the log message in line 112 is totally fine, but this log message doesn't help at all, does it?